### ISSUE TITLE
Can't login with Proton Drive

### ISSUE THREAD

**Issue by stemy2** [hwittenborn/celeste#208]
Hello,

I can't login with my proton account, it says "unable to authenticate. Check your credentials and try again", but my credentials are correct, i checked countless times but it keeps showing this error.

**Comment by elfenpiff** [comment]
same here with version v0.8.1 on archlinux

**Comment by bestlinuxgamers** [comment]
This is probably related to https://github.com/rclone/rclone/issues/7864

**Comment by bear454** [comment]
I was able to successfully log into proton drive today, on flatpak v0.8.3.

**Comment by Zero-33** [comment]
I was also able to login successfully yesterday, with flatpak v0.8.3

Also note, if you are using 2FA for your proton account, use the code that has only just changed giving you the longest amount of time (30secs or near), as the initial login did fail for me due to it taking awhile, resulting in that 2FA being out of date.

I was **unable** to login before when I was using the AUR packaged version 0.8.3-1. The reason it failed to login was linked to the 2FA box, I was unable to select it and so couldn't input the code. (no select box was present or could be seen for me to use)

**Comment by mclang** [comment]
Hmm... Doesn't work for me, tried several times.
Using Flatpak v0.8.3.

On another note, isn't this duplicate of https://github.com/hwittenborn/celeste/issues/167 ?

**Comment by humpfreybogart** [comment]
Same experience. cant login.  0.8.3 snapcraft on Ubuntu.
I also tried with 2 factor on.

**Comment by jgcharping** [comment]
I had the same problem on Linux Mint Debian.   I set up 2FA for the first time on Google Authenticator.   It still didn't work.    I checked my Proton Drive account info in my browser.   It had my Proton.me email address listed instead of the Email address that I used to sign up for and sign into my proton account.   I tried logging in with the proton.me email address and the 2FA and it worked first try.   It took a really long time, so i walked away for a while.   When I came back I was logged in.  I closed the app and opened it again and I'm still logged in.   Later I'll find out if my login survives a power cycle.

**Comment by SirMurloc** [comment]
I had this problem in with dual password mode turned on. When I switched to single password mode on Proton, I was able to log in.

**Comment by humpfreybogart** [comment]
> I had this problem in with dual password mode turned on. When I switched to single password mode on Proton, I was able to log in.

yes perhaps - but probably not working if you activate proton VPN.
on mine I can only use other VPN's than their own proton vpn, so I had to buy one more vpn solution. jsut so idiotic to deny your own vpn but allow others.

**Comment by quietusmaximusPM** [comment]
> yes perhaps - but probably not working if you activate proton VPN. on mine I can only use other VPN's than their own proton vpn, so I had to buy one more vpn solution. jsut so idiotic to deny your own vpn but allow others.

Ah, you saved me some serious grief trying to get this going.  It wouldn't have occurred to me otherwise. That said, I'm now awaiting sync check.  Baby steps, I guess, but thanks for the step forward.
